django_x509/base/admin.py

```diff
@@ -129,7 +128,7 @@ class Media:
 
     def get_urls(self):
         return [
-            url(r'^x509/ca/(?P<pk>[^/]+).crl$', self.crl_view, name='crl')
+            path('x509/ca/(?P<pk>[^/]+).crl', self.crl_view, name='crl')
```

Comment:
I think this will not work, `(?P<pk>[^/]+)` has to be converted to the new form of `path`, alternatively we can use `re_path`.